### PR TITLE
[Merged by Bors] - feat : Move various notions of uniform probability distributions to one file. 

### PR DESCRIPTION
**Pull request by JADekker**
feat : collect uniform distribution for measures
and uniform PMF in one file.

This collects the various notions of uniform
distribution in one file, to unify them with a
forthcoming PR.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Probability/Distributions/Uniform.lean [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 -/
+import Mathlib.Probability.Notation
+import Mathlib.Probability.Density
 import Mathlib.Probability.ProbabilityMassFunction.Constructions
 
-#align_import probability.probability_mass_function.uniform from "leanprover-community/mathlib"@"e50b8c261b0a000b806ec0e1356b41945eda61f7"
-
 /-!
-# Uniform Probability Mass Functions
+This file defines two related notions of uniform distributions, which will be unified in the future.
```
Comment (eric-wieser) [inline Mathlib/Probability/Distributions/Uniform.lean:11]: The first line of the docstring should usually be a `# Header` comment. Perhaps `Uniform distributions and probability mass functions`?

Comment (JADekker) [reply Mathlib/Probability/Distributions/Uniform.lean:12]: Thank you, done!

**Comment by Ruben-VandeVelde** [comment]
maintainer merge

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**